### DISCUSSION TITLE
用python 作疫情追蹤

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#619] [General]
https://fc.bnext.com.tw/articles/view/353?
這篇報導說明了用python連結我們現在學的爬蟲知識與技巧，為疫情作追蹤的實作架構。

架構主要是用爬蟲爬取確診者資料、足跡等資料，然後經由伺服器處理後，連結電子郵件發布系統，為大家發送疫情足跡電子郵件。

大家收到的疫情通知電子郵件就大概是這樣撰寫指令的，當然這篇報導不是鼓勵大家自己去建立程式，因為這樣涉及到許多個資與隱私的問題。

**Comment by leon10456** [comment]
好酷 馬上來自己做做看

**Comment by HsuChen88** [comment]
謝謝分享 可以學習用此方法做別的事情

**Comment by awkward-willy** [comment]
爬蟲可以運用的範圍真的很廣呢
畢竟要透過它才能大量獲得資料以便後續的利用與分析
真的是我們必學的技能之一
也可見老師的課能讓人獲益良多阿

**Comment by sys511613** [comment]
爬蟲真的是有好多用處
文章中講到的發送電子郵件感覺也蠻有趣的

**Comment by Jimmy911218** [comment]
爬蟲真的能提供便利超多的
